releasenotes/notes/decompose-status-further-with-subcommands-8fd7f1282e990449.yaml

```diff
@@ -7,4 +7,6 @@
 # Each section note must be formatted as reStructuredText.
 ---
 enhancements:
```

Comment:
These are reStructured text - I think you may need "``" instead of "`" to delineate code

Reply:
Thanks ! Good to know :)